app/src/playStore/java/org/fossasia/openevent/general/search/GeoLocationUI.kt

```diff
@@ -17,15 +20,34 @@ import org.fossasia.openevent.general.MainActivity
 import timber.log.Timber
 import java.util.*
 
+private const val TO_SEARCH: String = "ToSearchFragment"
+
 class GeoLocationUI {
     var query = ""
```

Comment:
This shouldn't be needed now

Reply:
It is actually in the way I did this. onLocationResult stores the result after the activity is loaded and when the current location is clicked it calls the method and sees if the query is empty or not and sets the search parameter. If it's not global then it will remain empty.

Reply:
Done